.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "semi": true,
```

Comment:
🐘  Just realised I left out the `$schema`: `"$schema": "http://json.schemastore.org/prettierrc",`